### PR TITLE
#96 Improve command line parser performance.

### PR DESCRIPTION
**Pull request by pzheng1025**
Use hashmap to store mount options to reduce search to O(1)

### CODE REVIEW

File: fuse_ll/src/fuse/mount.rs [hunk 1]
```diff
@@ -148,13 +157,12 @@ mod param {
...
-                let mount_options = get_mount_options();
-                let option = mount_options
-                    .iter()
-                    .find(|x| x.regex.is_match(&op))
-                    .unwrap();
-                (option.parser)(&mut args, &option, &op)
+            let mount_options_map = super::get_mount_options_map();
+            options.iter().for_each(|op| {
+                let key = op.split('=').collect::<Vec<_>>()[0].to_string();
+                if let Some(option) = mount_options_map.get(&key) {
```
Comment (rogercloud) [inline fuse_ll/src/fuse/mount.rs:163]: Will we report error when the option is not defined?

File: fuse_ll/src/fuse/mount.rs [hunk 2]
```diff
@@ -283,13 +289,12 @@ mod param {
...
```
Comment (rogercloud) [inline fuse_ll/src/fuse/mount.rs:295]: The same as above

File: fuse_ll/src/fuse/mount.rs (hunk 1 above)
Comment (pzheng1025) [reply fuse_ll/src/fuse/mount.rs:163]: All the options that reach here will be valid options.
Option validation will be done by command line parser.

Comment (rogercloud) [reply fuse_ll/src/fuse/mount.rs:163]: If it's garanteed to be validate, `unwrap` is a clear way I think.

Comment (pzheng1025) [reply fuse_ll/src/fuse/mount.rs]: Modified. Thanks.

**Pull request merged**